test/parallel/test-http-localaddress.js

```diff
@@ -28,26 +28,26 @@ if (!common.hasMultiLocalhost())
 const http = require('http');
 const assert = require('assert');
 
-const server = http.createServer(function(req, res) {
+const server = http.createServer((req, res) => {
   console.log(`Connect from: ${req.connection.remoteAddress}`);
   assert.strictEqual(req.connection.remoteAddress, '127.0.0.2');
 
-  req.on('end', function() {
+  req.on('end', () => {
     res.writeHead(200, { 'Content-Type': 'text/plain' });
     res.end(`You are from: ${req.connection.remoteAddress}`);
   });
   req.resume();
 });
 
-server.listen(0, '127.0.0.1', function() {
+server.listen(0, '127.0.0.1', () => {
   const options = { host: 'localhost',
-                    port: this.address().port,
+                     port: server.address().port,
```

Comment:
@tpanthera - once more issue here - you put one extra space in line 44 before the code. Align it to the previous line, and I guess we are all good here.

Reply:
Thanks for correcting, committed the revised code.